ci/jenkins/test.sh

```diff
@@ -549,11 +572,14 @@ function deliver_antrea {
         DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --build-tag ${BUILD_TAG} --pull
         IMG_TAG="${BUILD_TAG}" ./hack/generate-manifest.sh $MANIFEST_ARGS > build/yamls/antrea.yml
     else
-        DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
+        if [[ $TESTBED_TYPE == "kind" || $TESTBED_TYPE == "kind-flexible-ipam" ]]; then
+           DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh 
```

Comment:
Why was the `--pull` option removed when running build-antrea-linux-all.sh? The jenkins agent node shouldn't be affected by the rate limit issue, as we've logged into the Docker account in https://github.com/antrea-io/antrea/blob/48ce6311abc0ae5ace498419d57fd807407f2086/ci/jenkins/jobs/macros.yaml#L216.